glide-core/src/client/mod.rs

```diff
@@ -485,24 +485,24 @@ impl Client {
         password: Option<String>,
         re_auth: bool,
     ) -> RedisResult<Value> {
+        match self.internal_client {
+            ClientWrapper::Standalone(ref mut client) => {
+                client.update_connection_password(password.clone()).await?;
+            }
+            ClientWrapper::Cluster { ref mut client } => {
+                client.update_connection_password(password.clone()).await?;
+            }
+        }
         if re_auth {
             let routing = RoutingInfo::MultiNode((
                 MultipleNodeRoutingInfo::AllNodes,
```

Comment:
Good to add to docs that client re-auth on all nodes

Reply:
I'll take care of it. Do you think it makes sense that it would do different? Seems clear to me that this is the impl.

Reply:
That's obvious, but good to have

Reply:
Done.